tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -300,13 +302,20 @@ std::map<std::string, ShortCircuitVoltageScaling> const sc_voltage_scaling_mappi
 using CalculationFunc =
     std::function<BatchParameter(MainModel&, CalculationMethod, Dataset const&, ConstDataset const&, Idx)>;
 
+std::map<std::string, OptimizerStrategy> const optimizer_strategy_mapping = {
```

Comment:
sonar cloud suggestion

```suggestion
std::map<std::string, OptimizerStrategy, std::less<>> const optimizer_strategy_mapping = {
```